src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt

```diff
@@ -1167,6 +1168,35 @@ private fun RsStubLiteralType?.serialize(dataStream: StubOutputStream) {
     }
 }
 
+
+class RsAssocTypeBindingStub(
+    parent: StubElement<*>?, elementType: IStubElementType<*, *>,
+    override val name: String?
+) : StubBase<RsAssocTypeBinding>(parent, elementType),
+    RsNamedStub {
+
+    object Type : RsStubElementType<RsAssocTypeBindingStub, RsAssocTypeBinding>("ASSOC_TYPE_BINDING") {
+        override fun deserialize(dataStream: StubInputStream, parentStub: StubElement<*>?) =
+            RsAssocTypeBindingStub(parentStub, this,
+                dataStream.readNameAsString()
+            )
+
+        override fun serialize(stub: RsAssocTypeBindingStub, dataStream: StubOutputStream) =
+            with(dataStream) {
+                writeName(stub.name)
+            }
+
+        override fun createPsi(stub: RsAssocTypeBindingStub): RsAssocTypeBinding =
+            RsAssocTypeBindingImpl(stub, this)
+
+        override fun createStub(psi: RsAssocTypeBinding, parentStub: StubElement<*>?) =
+            RsAssocTypeBindingStub(parentStub, this, psi.identifier.text)
+
+
+        override fun indexStub(stub: RsAssocTypeBindingStub, sink: IndexSink) {}
```

Comment:
You can drop this method because `RsStubElementType` already has default empty implementation